modular_skyrat/modules/modular_implants/code/soulcatcher/soulcatcher_component.dm

```diff
@@ -220,7 +222,6 @@ GLOBAL_LIST_EMPTY(soulcatchers)
 		var/message_to_log = "[key_name(new_soul)] joined [src] inside of [parent_atom] at [loc_name(soulcatcher_turf)]"
 		parent_atom.log_message(message_to_log, LOG_GAME)
 		new_soul.log_message(message_to_log, LOG_GAME)
-
```

Comment:
Leave that there, it helps readability. I can't make suggestions on removed lines, unfortunately.